topic_resources/capstone/presentation.md

```diff
@@ -1,14 +1,13 @@
 # Capstone Presentation
-The capstone presentation highlights your application as the culmination of your work at Ada. 
-It should be <b>no more than 5 minutes</b> in length and should have the following components: 
+The capstone presentation highlights your application as the culmination of your work at Ada.
+It should be __no more than 5 minutes__ in length and should have the following components: 
 
 - What problem does the app solve? Who does it solve it for?
 - What are 1-2 major features of the application, demoed from the eyes of a user?
-- What was one technical challenge you overcame? 
+- What was one technical challenge you overcame?
 
 Prior to demoing, you should have the following:
 - Provide URLs to the first person presenting in your group; ensure you have credentials for accounts accessible
```

Comment:
We ended up not doing this at demos for C3. Everyone showed from their own laptop. It was moderately obnoxious, but with proper prep works just fine. Dunno if we wanna do it that way again or not. One thing I'd add would be a requirement for folks with hardware or props or whatevs to clear the space before the next person goes on.